etl/steps/data/garden/oecd/2024-07-01/road_accidents.py

```diff
@@ -0,0 +1,149 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+import owid.catalog.processing as pr
+import pandas as pd
+from owid.catalog import Origin, Table
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+new_data_cols = [
+    "accident_deaths",
+    "accident_injuries",
+    "accidents_with_injuries",
+    "deaths_per_million_population",
+    "passenger_kms_rail",
+    "passenger_kms_road",
+    "passenger_kms_car",
+    "passenger_kms_bus",
+    "deaths_per_billion_kms",
+]
+
+
+def add_origins(tb: Table, cols: list, origins: Origin) -> Table:
+    for col in cols:
+        tb[col].origins = origins
+    return tb
+
+
+def check_road_passenger_travel(tb_row: pd.Series):
+    """Check if road passenger travel is identical to bus or car passenger travel. If so, remove road passenger travel."""
+    road_passengers = tb_row["passenger_kms_road"]
+    # if road passenger travel is NA or both bus and car passenger travel are NA, there is nothing to compare
+    if pd.isna(road_passengers) or (pd.isna(tb_row["passenger_kms_bus"]) & pd.isna(tb_row["passenger_kms_car"])):
+        return road_passengers
+    # check whether road passenger travel is identical to bus travel
+    elif not pd.isna(tb_row["passenger_kms_bus"]) and tb_row["passenger_kms_bus"] == road_passengers:
+        return None
+    # check whether road passenger travel is identical to car travel
+    elif not pd.isna(tb_row["passenger_kms_car"]) and tb_row["passenger_kms_car"] == road_passengers:
+        tb_row["passenger_kms_road"] = None
+        return None
+
+
+def choose_latest_data(tb_row: pd.Series, old_col: str, new_col: str):
+    if pd.isna(tb_row[new_col]):
+        return tb_row[old_col]
+    else:
+        return tb_row[new_col]
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = paths.load_dataset("road_accidents", channel="meadow", version="2024-07-01")
```

Comment:
I personally think it looks confusing if I am importing another garden step thats named the same - I prefer the clarity of the version, even if it is more of a hassle to update. But I will look into seperating the historic data in individual steps, so maybe this will not even be necessary in the end.